packages/pigeon/example/app/ios/Runner/AppDelegate.swift

```diff
@@ -25,6 +25,16 @@ private class PigeonApiImplementation: ExampleHostApi {
     }
     completion(.success(true))
   }
+
+  func sendMessageModernAsync(message: MessageData) async throws -> Bool {
+    try? await Task.sleep(nanoseconds: 2_000_000_000)
```

Comment:
is this used for unit test? we shouldn't wait for so long as it slows down the tests

Reply:
This is used in example app

Reply:
Removed `Task.sleep`